src/app/dashboard/layout.tsx

```diff
@@ -15,22 +15,24 @@ export default function DashboardLayout({
   children: Readonly<React.ReactNode>;
 }) {
   return (
-    <div className="min-h-screen">
-      <header className="flex justify-between p-4">
-        <nav className="flex items-center gap-8">
-          <AppLogo />
-          <Link href="/dashboard">Dashboard</Link>
-          <Link href="/dashboard/events">Wydarzenia</Link>
-          <Link href="/dashboard/settings">Ustawienia konta</Link>
-        </nav>
-        <div className="flex gap-4">
-          <ThemeSwitch />
-          <AuthButton />
-        </div>
-      </header>
-      <main className="flex min-h-[calc(100vh-80px)] flex-col px-4 py-8">
-        {children}
-      </main>
+    <div className="flex min-h-screen justify-center">
+      <div className="container">
```

Comment:
Niepotrzebny flexbox tutaj
```suggestion
    <div className="min-h-screen">
      <div className="container mx-auto">
```